src/dms/backend/EdFi.DataManagementService.Backend.Postgresql/Operation/UpsertDocument.cs

```diff
@@ -19,8 +19,7 @@ public interface IUpsertDocument
     public Task<UpsertResult> Upsert(
         IUpsertRequest upsertRequest,
         NpgsqlConnection connection,
-        NpgsqlTransaction transaction,
-        TraceId traceId
+        NpgsqlTransaction transaction
```

Comment:
Problem where we were passing in traceId again, although already in request object. Led to some test problems.